src/Stoa.ts

```diff
@@ -586,6 +587,72 @@ class Stoa extends WebService
             );
     }
 
+    /**
+     * POST /utxos
+     *
+     * Returns UTXO's information about the UTXO hash array.
+     *
+     * Returns a array of UTXO's information.
+     */
+    private getUTXOs (req: express.Request, res: express.Response)
```

Comment:
It would be great if the request parameter description is added.
`The array of UTXO hash`

Reply:
I fixed